01_introduction_to_algorithms/csharp/01_binary_search/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I don't know much about Objective-C, are these files in `.vscode` needed? It looks like an auto-generated directory to me.